compliance/github/available_seats/available_seats.pt

```diff
@@ -1,6 +1,6 @@
 name "GitHub.com Available Seats Report"
 rs_pt_ver 20180301
-short_description "Gets the number of available seats for a licensed GitHub Org and creates an incident if they are out of the policy range.  See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/github/available_seats) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+short_description "Gets the number of available seats for a licensed GitHub Org and creates an incident if they are out of the policy range.  See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/github/available_seats) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
 long_description "Version: 1.4"
```

Comment:
Do you need to bump the version to have this take effect? Not sure what the version is used for in that regard

Reply:
yes, I didn't actually need to change this one so I didn't change the version or changelog.